### ISSUE TITLE
Unable to run DBITail

### ISSUE THREAD

**Issue by adrianolabate** [pschweitz/DBITail#2]
Hello,

I am unable to run dbi-tail on my pc. 

When I run it with the command "java -jar dbi-Tail.jar", I have the following output:

```
août 17 2016 CEST 14:50:50.381 [JavaFX Application Thread] INFO com.dbiservices.tools.Logger - LogLevel: Debug
Exception in Application start method
java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:389)
        at com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:328)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at sun.launcher.LauncherHelper$FXHelper.main(Unknown Source)
Caused by: java.lang.RuntimeException: Exception in Application start method
        at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:917)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$155(LauncherImpl.java:182)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NumberFormatException: For input string: " server.log"
        at java.lang.NumberFormatException.forInputString(Unknown Source)
        at java.lang.Integer.parseInt(Unknown Source)
        at java.lang.Integer.parseInt(Unknown Source)
        at com.dbiservices.monitoring.tail.DbiTail.initTreeNodes(DbiTail.java:1293)
        at com.dbiservices.monitoring.tail.DbiTail.start(DbiTail.java:185)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$162(LauncherImpl.java:863)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$175(PlatformImpl.java:326)
        at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
        at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
        at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
        ... 1 more
Exception running application com.dbiservices.monitoring.tail.DbiTail
```

My pc is on Windows 7. 
Locale is fr-ch.
My Java version is:

```
java version "1.8.0_101"
Java(TM) SE Runtime Environment (build 1.8.0_101-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.101-b13, mixed mode)
```

dbi-tail is the last version

Any suggestion to resolve that issue?

**Comment by pschweitz** [comment]
Hi Adrian,

 You likely have a corrupted "default.tree" file in the "etc" folder.

Could you try to rename that file to "default_old.tree" for instance and try again?

**Comment by adrianolabate** [comment]
Hello Philippe,
Thanks for your answer. 
I think to have found what causes corrupted file. If you rename your entry and add a comma "," the file cannot be read anymore. 
For example "WildFly server.log" is OK, but if I rename it "WildFly, server.log" it will fail the next time.
Regards

**Comment by pschweitz** [comment]
Hi Adriano,

I corrected the issue, and pushed a version 1.2.3.

Best regards,
Philippe
